hw/top_earlgrey/cdc/cdc_waivers.w_fanout.tcl

```diff
@@ -0,0 +1,15 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Verix CDC waiver file
+
+set_rule_status -rule {W_FANOUT} -expression {Driver =~ "*u_lc_ctrl*.u_prim_lc_sender_escalate_en*"} -status {Waived} -comment {No Reconvergence issue. Each IP handles Escalate En individually; set_rule_status run by  user root on host smart-dis-us-central1-ac9805fe-454c-4151-83d0-2daae3aac116 at Monday, 14 November 2022 22:54:33 PDT}
```

Comment:
can you remove the auto-added parts here that contain the smart ID? there are other cases in this PR as well.